bot/cogs/bot.py

```diff
@@ -60,6 +62,18 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        """
+        Auto codeblock python code
+        """
+        if msg.content.count("\n") >= 3:  # more than three lines
```

Comment:
Didn't we agree that it was over 3?

Reply:
Three or more counts of `\n` should mean over 3 lines, unless the user includes trailing newlines.

Reply:
Ahh yeah, my bad